plugins/catalog-react/src/components/EntityDisplayName/EntityDisplayName.tsx

```diff
@@ -77,7 +81,7 @@ export const EntityDisplayName = (
   );
 
   // The innermost "body" content
-  let content = <>{primaryTitle}</>;
+  let content = <div className={classes.breakTextOverflow}>{primaryTitle}</div>;
```

Comment:
I think we have a problem actually. This feature requires a CSS block context, while this component requires to function in inline contexts. Thinking more about this, I think maybe EntityDisplayName itself cannot support this feature at all, but rather, the surrounding div that the display name is rendered in, needs to be the one with the overflow handling. Does that make sense?

Reply:
@freben I have made the changes requested by you .